### DISCUSSION TITLE
Eurang팀 개발 방식 논의

### DISCUSSION THREAD

**Discussion by judemin** [euphony-io/euphony#68] [General]
upstream 리포지토리에 직접 push하거나 따로 branch를 만드는 것은 어려울 것이라 판단해서
매번 테스트 코드를 작성하고 이를 공유하기 위해 @kuro11pow2 님의 리포지토리로 각자의 리포지토리에서 PR을 요청해서
@kuro11pow2 님의 리포지토리를 git remote를 통해 새로운 스트림으로 등록하고 upstream 리포지토리에 직접 PR을
요청할 수 있을 때까지 작업할 예정입니다.

![image](https://user-images.githubusercontent.com/20484492/135261251-d2c98fb8-c52f-4a99-8d16-d55d67e7f563.png)

**Comment by seohy02** [comment]
굳굳굳 !!!

**Comment by kuro11pow2** [comment]
좋아요! develop branch를 따로 만들어서 해봐요!

**Reply by judemin** [reply]
넵 그럼 브랜치 업데이트 하겠습니다!

**Reply by kuro11pow2** [reply]
브랜치 올려뒀습니다!

**Comment by aiclaudev** [comment]
좋네요 감사합니다~!

**Comment by ycs1m1yk** [comment]
좋습니다ㅎㅎ
